docs/Contributing/API-for-contributors.md

```diff
@@ -2754,11 +2836,35 @@ If both `team_id` and `team_name` parameters are included, this endpoint will re
 
 #### Example
 
-`POST /api/v1/fleet/mdm/scripts/batch`
+`POST /api/v1/fleet/scripts/batch`
 
 ##### Default response
 
-`204`
+`Status: 204`
+
+## Software
+
+### Batch-apply software 
+
+_Available in Fleet Premium_
 
-<meta name="pageOrderInSection" value="800">
-<meta name="description" value="Read about Fleet API routes that are helpful when developing or contributing to Fleet.">
+`POST /api/v1/fleet/software/batch`
+
+#### Parameters
+
+| Name      | Type   | In    | Description                                                                                                                                                           |
+| --------- | ------ | ----- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| team_id   | number | query | The ID of the team to add the software package to. Only one team identifier (`team_id` or `team_name`) can be included in the request, omit this parameter if using `team_name`. |
+| team_name | string | query | The name of the team to add the software package to. Only one team identifier (`team_id` or `team_name`) can be included in the request, omit this parameter if using `team_id`. |
+| dry_run   | bool   | query | Validate the provided software packages and return any validation errors, but do not apply the changes.                                                                         |
+| software  | array  | body  | An array of objects containing `url` to the software package (PKG, MSI, EXE or DEB),`install_script` - command that Fleet runs to install software, `pre_install_query` - condition query that determines if the install will proceed, and `post_install_script` that runs after software install. |
```

Comment:
I think we usually say "list" instead of "array". Updating this to match changes to software parameter above:
```suggestion
| software  | list   | body  | An array of software objects. Each object consists of:`url`- URL to the software package (PKG, MSI, EXE or DEB),`install_script` - command that Fleet runs to install software, `pre_install_query` - condition query that determines if the install will proceed, and `post_install_script` - script that runs after software install.   |